app/data/jsonSchemaForm/validationSchemas.ts

```diff
@@ -7,6 +8,8 @@ import operatorSchema from "./operatorSchema";
 const validationSchemas = {
   project_contact: projectContactSchema,
   project_manager: projectManagerSchema,
+  //brianna --why is it not project_reporting_requirement?
```

Comment:
because that's the name of the table

Reply:
Sorry, that was a confusing way to ask that question. I mean I'm not sure I'm using the `project` prefix right. I called all my variables `projectReportingRequirement`, `projectQuarterlyReports`, (vs. `reportingRequirement, quarterlyReports) because I figured the prefix meant the report was tied to a project (like how projectContacts are just for a project, but contacts are their own page). But if the name is supposed to be coming from the table, then maybe I should change it?

Reply:
Yes, I suppose that the `project` prefix should be removed when there is no ambiguity.

Reply:
Although one could think that naming a component `QuarterlyReportForm` is confusing, if all of the other project forms start with `Project`

Reply:
Yes, that's where I was initially coming from. If you're only looking at front-end files, it looks inconsistent. Perhaps I'll leave it as-is then?

Reply:
Yeah I think we can leave as-is for now